share/cmake/modules/FindOpenShadingLanguage.cmake

```diff
@@ -0,0 +1,80 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
OSL 1.11+ has proper exported cmake configuration files that will set up all these targets. Nobody should need a FindOpenShadingLanguage.cmake.

Reply:
The cmake method also provides a support to link against a custom OSL version using the cmake variable `OpenShadingLanguage_ROOT`. For now that's the only way for OCIO to access OSL.